### PR TITLE
Add `chain` to RB commands

### PR DESCRIPTION
**Pull request by tsnewnami**
addresses: https://github.com/drinkcoffee/arbitrage-bot/issues/10

### CODE REVIEW

File: bin/rb/src/commands/chain.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    println!("Latest block number: {latest}");
+    Ok(())
+}
+
+pub async fn finalized(provider: RootProvider) -> Result<()> {
+    let finalized = provider
+        .get_block(BlockId::finalized(), Default::default())
+        .await
+        .wrap_err("failed to get finalized block")?
+        .ok_or(eyre!("finalized block is still pending"))?;
```
Comment (sergerad) [inline bin/rb/src/commands/chain.rs:35]: Does the final eyre! error not ignore the previous err and wrapped err?

File: bin/rb/src/main.rs [hunk 2]
```diff
@@ -37,22 +38,26 @@ impl Cli {
...
-            Some(PoolCommands::Dump) => pool_tick_dump_command().await,
-            Some(PoolCommands::Info) => pool_tick_info_command().await,
-            None => unreachable!(),
+            PoolCommands::TickSpacing => pool_tick_spacing().await,
+            PoolCommands::CurrentTick => pool_current_tick_command().await,
+            PoolCommands::Dump => pool_tick_dump_command().await,
+            PoolCommands::Info => pool_tick_info_command().await,
+        },
+        (Commands::Chain(args), provider) => match args.command {
+            commands::ChainCommands::ID => chain_id(provider).await,
```
Comment (sergerad) [inline bin/rb/src/main.rs:57]: Some command fns have _command suffix and some don't. Prefer without suffix but happy either way as long as all fns do the same

**Review by sergerad — APPROVED** [review]

**Review by sergerad — APPROVED** [review]

File: bin/rb/src/commands/chain.rs (hunk 1 above)
Comment (EdwardJES) [reply bin/rb/src/commands/chain.rs:44]: going to remove the wrapped error for now.

Comment (tsnewnami) [reply bin/rb/src/commands/chain.rs:35]: Going to remove the wrap err for now.

**Pull request merged**